docker/openemr/5.0.2/openemr.conf

```diff
@@ -1,40 +1,33 @@
 LoadModule rewrite_module modules/mod_rewrite.so
 LoadModule allowmethods_module modules/mod_allowmethods.so
 
-
-# The ServerName directive sets the request scheme, hostname and port that
-# the server uses to identify itself. This is used when creating
-# redirection URLs. In the context of virtual hosts, the ServerName
-# specifies what hostname must appear in the request's Host: header to
-# match this virtual host. For the default virtual host (this file) this
-# value is not decisive as it is used as a last resort host regardless.
-# However, you must set it for any further virtual host explicitly.
-#ServerName www.example.com
-
-
 ## Security Options
+# Strong HTTP Protocol
+HTTPProtocolOptions Strict
```

Comment:
Not sure what this does, but can always try it out. @danehrlich1 , this won't force https, correct?